fe/src/app/core/core.module.ts

```diff
@@ -2,16 +2,19 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
```

Comment:
Routing isn't working correctly, need to add the following line: `import { RouterModule } from '@angular/router';`. This will suffice at least until a better option comes along.